ios/Plugin/Plugin.swift

```diff
@@ -246,5 +246,37 @@ public class CameraPreview: CAPPlugin {
             call.reject("failed to set flash mode")
         }
     }
+
+        @objc func startRecordVideo(_ call: CAPPluginCall) {
+            DispatchQueue.main.async {
+
+            let quality: Int? = call.getInt("quality", 85)
+            
+            self.cameraController.captureVideo { (image, error) in
+
+                guard let image = image else {
+                    print(error ?? "Image capture error")
+                    guard let error = error else {
+                        call.reject("Image capture error")
+                        return
+                    }
+                    call.reject(error.localizedDescription)
+                    return
+                }
+
+                    self.videoUrl = image
```

Comment:
where is this variable declared? I get compilation error in xcode now.

Reply:
same for me `Value of type 'CameraPreview' has no member 'videoUrl'`

Reply:
All errors occurring  with Video functions during Swift compile. I thing contributors should roll back or quick fix needed

Reply:
I'm getting the same error durign swift compile.

Reply:
why has this been merged into master? This breaks the build!

Reply:
No hope to get the video items working @rudyZanotti? Would be great, but also see the same errors.